ksql-functional-tests/src/test/resources/query-validation-tests/joins.json

```diff
@@ -1,8 +1,8 @@
 {
   "tests": [
     {
-      "comment": "re-enable after KIP-479 is merged",
-      "enabled": false,
+      "comment": "Tests covering the use of Joins",
```

Comment:
nit: would you mind just removing the `comment` and `enabled` fields from the JSON for each of these please?

Reply:
ack